src/opening-period/page/CreateNewOpeningPeriodPage.scss

```diff
@@ -16,45 +16,26 @@
   label {
     display: block;
     margin-top: $spacing-m;
+    font-family: HelsinkiGrotesk-Medium, Arial, sans-serif;
```

Comment:
Would it make sense to put this into index.scss as a global style

Reply:
I can try that. It might be challenging because often HDS styles overrides our styles that are not specific enough. But I'll see what we can do. 

- [ ] Check if this could be put into index.scss as a global style

Reply:
I thought that HDS did not style labels, and that is why we needed to have our styles for them 🤔

Reply:
I put it to index.scss. Looked UI locally and it felt normal, so let's go with that!